zipper.go

```diff
@@ -71,9 +71,10 @@ func NewZipper(name string, meshConfig map[string]config.Downstream, options ...
 			core.WithNonBlockWrite(),
 			core.WithConnectUntilSucceed(),
 		)
-		downstream := core.NewClient(name, core.ClientTypeUpstreamZipper, clientOptions...)
+		downstream := core.NewClient(downstreamName, core.ClientTypeUpstreamZipper, clientOptions...)
```

Comment:
The downstream name bug is here @fanweixiao @wujunzhuo 

Now change `name` to `downstreamName`